app/models/user.rb

```diff
@@ -658,6 +658,8 @@ def destroy_follows
   end
 
   def index_roles(_role)
+    return unless persisted?
```

Comment:
This change is explained in detail here https://github.com/forem/forem/pull/10313#issuecomment-691849151